exconf/utils.py

```diff
@@ -55,8 +55,13 @@ def get_logger(logger_name="magine-services"):
 LOG = get_logger()
 
 
-def read_yaml(file_path):
-    return yaml.load(open(file_path).read())
+def read_yaml(file_path, out=sys.stdout):
+    try:
```

Comment:
This will now swallow the exceptions? I think it's better to raise a descriptive error than to just return None in case of error.